src/s3/objectCreated.ts

```diff
@@ -0,0 +1,41 @@
+import { S3CreateEvent, S3Handler } from 'aws-lambda';
+import dependencies, { Container } from '../dependencies';
+
+const isMetadataFile = (key: string) => key.endsWith('.json');
+const getDocumentIdFromKey = (key: string) => key.split('/')[0];
```

Comment:
Do we want the documents to be in ID/ID? Cause there was talk about separating them into customer folders (?) (maybe I misunderstood tho). So like customerID/documentID, in which case you'd probably want [1]?

Reply:
Good point, I made the assumption that this would have document id first. We did speak about it but I don't know if we decided 100% on the ordering. I guess the problem we have right now is that we are not mandating _any_ information in the metadata that is provided so we can't guarantee a customer id will always be present.

Reply:
That's fair 😁